pkg/loki/topology_query.go

```diff
@@ -50,6 +56,13 @@ func NewTopologyQuery(cfg *Config, start, end, limit, rateInterval, step, metric
 	case "droppedBytes":
 		f = "rate"
 		t = "PktDropBytes"
+	case "dnsLatencies":
+		f = "avg_over_time"
+		t = "DnsLatencyMs"
+	case "flowRtt":
+		f = "avg_over_time"
+		t = "TimeFlowRttNs"
+		factor = 0.000001 // nanoseconds to miliseconds
```

Comment:
to avoid any floating point precision issue, it could be better to divide by an int rather than multiply by a decimal
Also, as long as it's only used to inject in a query (as string), that could be written right away as a string (if I'm correct we don't use the numerical value for anything) - that would also avoid that `strings.Replace` done below to get the right format for loki :-)

Reply:
sure, a string would be better as it would keep both operator and value at once :+1:

Reply:
https://github.com/netobserv/network-observability-console-plugin/pull/365/commits/82f4591dcb07b0449a9cb8cb3875c6bd702a853b